Ginger/GingerCoreNET/Database/IDatabase.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Amdocs.Ginger.CoreNET
+{
+    public interface IDatabase
+    {
+        Boolean OpenConnection(Dictionary<string, string> parameters);
+        void CloseConnection();
+        List<string> GetTablesList(string Name= null);// Keyspace - cassandra ??
+        List<string> GetTablesColumns(string table);
+        string RunUpdateCommand(string updateCmd, bool commit = true);
+        string GetSingleValue(string Table, string Column, string Where);
+        List<object> DBQuery(string Query); //  int? timeout = null : TODO // Return Datatable 
```

Comment:
return value we need something like table but for no sql we need to disucss

Reply:
done